packages/main/src/themes/Panel.css

```diff
@@ -81,7 +81,7 @@
 }
 
 .ui5-panel-content {
-	padding: 0.625rem 1rem 1.375rem 1rem;
+	padding: 0.625rem 1rem 0.625rem 1rem;
```

Comment:
padding: 0.625rem 1rem; would be enough.

Reply:
Done